spyder/app/mainwindow.py

```diff
@@ -1151,16 +1166,17 @@ def __getattr__(self, attr):
             pass
         return super().__getattr__(attr)
 
-    def update_lsp_logs(self):
-        """Create an action for each lsp log file."""
-        self.menu_lsp_logs.clear()
-        lsp_logs = []
+    def update_debug_logs(self):
+        """Create an action for each lsp and debug log file."""
+        self.menu_debug_logs.clear()
+        debug_logs = []
         files = glob.glob(osp.join(get_conf_path('lsp_logs'), '*.log'))
+        files += glob.glob(os.environ['SPYDER_DEBUG_FILE'])

```

Comment:
Is there a need to use `glob.glob` on this line if `SPYDER_DEBUG_FILE` is a file path and not a directory?

Reply:
No, there is not a need. I'll remove glob